gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -127,7 +127,32 @@ elif [[ "${RE_JOB_SCENARIO}" == "ironic" ]]; then
   echo "export DEPLOY_IRONIC=yes" >> /opt/rpc-openstack/RE_ENV
 fi
 
-# prepare rpc-o configs
+# generate infra1 deploy script
+cat > /opt/rpc-openstack/deploy-infra1.sh <<EOF
+#!/bin/bash
+set -evu
+# starts the deploy from infra1 vm
+source /opt/rpc-openstack/RE_ENV
+pushd /opt/rpc-openstack
+  scripts/deploy.sh
+popd
+pushd /opt/rpc-openstack/playbooks
+  openstack-ansible -i 'localhost,' \
+                    -e 'apt_target_group=localhost' \
+                    -e "apt_artifact_enabled='${RPC_APT_ARTIFACT_ENABLED}'" \
+                    -e "apt_artifact_mode='${RPC_APT_ARTIFACT_MODE}'" \
+                    site-artifacts.yml
+  openstack-ansible openstack-ansible-install.yml
+popd
+pushd /opt/openstack-ansible/scripts
+  python pw-token-gen.py --file /etc/openstack_deploy/user_secrets.yml
+popd
+pushd /opt/openstack-ansible/playbooks
+  openstack-ansible setup-everything.yml
+popd
+EOF
+
+# sync rpc-o to infra prepare rpc-o configs and
```

Comment:
Was something supposed to be after the "and"?

Reply:
Yeah, good catch, just needed to be moved back a bit.